WebPortal/Pages/Admin/Administration.razor

```diff
@@ -3,8 +3,8 @@
 
 @inject IUserInformationService _userInformationService
 @inject ServiceAuthManager _serviceAuthManager
-@page "/administration"
-@page "/admin"
+@page "/administration/old"
```

Comment:
Why "old", or you want to keep this URL temporarily? 
Old and New are troublesome words.

Reply:
It's not meant to be used, and there's no link or anything to it. I had used the /old urls while developing the new version. I'm thinking in case I missed anything we can keep the file. Maybe I'll remove the page mapping, or maybe just delete the unused stuff entirely and if we ever need it we can restore it from git history.